### DISCUSSION TITLE
Witch water help

### DISCUSSION THREAD

**Discussion by skytheraiders** [NovaMachina-Mods/ExNihiloSequentia#392] [Q&A]
I'm trying to get the barrel to make witch water consistently but after a while, the mycelium changes back into dirt which breaks the automation. Do you have any suggestions?

**Comment by NovaMachina** [comment]
Which version are you using? The latest version should have this fix.

**Reply by NovaMachina** [reply]
Actually I just realized that there was a build failure causing the latest changes to not get pushed. I'll get this fixed soon.

**Comment by NovaMachina** [comment]
This should now be fixed in an update that will be live in a few minutes.
